src/account/AccountList.js

```diff
@@ -18,16 +18,20 @@ export default class AccountList extends React.PureComponent {
 
     return (
       <View>
-        {accounts.map((account, i) =>
-          <AccountItem
-            key={account.apiKey}
-            index={i}
-            showDoneIcon={i === 0 && auth.apiKey !== '' && auth.apiKey === account.apiKey}
-            {...account}
-            onSelect={onAccountSelect}
-            onRemove={onAccountRemove}
-          />,
-        )}
+        <FlatList
+          keyboardShouldPersistTaps="always"
+          data={accounts}
+          keyExtractor={item => `${item.email}${item.realm}`}
+          renderItem={({ item, index }) =>
+            <AccountItem
+              key={`${item.email}${item.realm}`}
```

Comment:
We don't need a `key` property on the child item.
I saw we use it in few other places where we do have FlatList you can remove it from these places too.